cpp/src/io/json/experimental/read_json.cpp

```diff
@@ -72,27 +91,9 @@ size_type find_first_delimiter_in_chunk(host_span<std::unique_ptr<cudf::io::data
   auto const buffer = ingest_raw_input(sources,
```

Comment:
Note: `find_first_delimiter_in_chunk` is used only in unit test `json_chunked_reader.cpp`.